CONTRIBUTING.md

```diff
@@ -1,4 +1,47 @@
-# Contributing
+# NAVODILA ZA OSNOVNO UPORABO NADLOGARJA
```

Comment:
Po mojem ni treba, da je vse skupaj v velikih začetnicah, itak imamo že `#` in `##`.